### ISSUE TITLE
GQL - Error on mutations and submission queries when using repeater fields

### ISSUE THREAD

**Issue by danieladarve** [verbb/formie#299]
**Description**
Consider the following simple form (1 email field, and 1 repeater field with a single line text field)
[Form](https://imgur.com/9Srpxl2)

If I submit the form using a twig generated form, it works as expected no problems. You can see the submission and all the field values.

However, when you query the submissions via the GQL explorer:
```
{
  formieSubmissions {
    ... on contact_Submission {
      id
      emailAddress
      repeater
    }
  }
}
```
In the schema definition, it shows the repeater field as type **String**

You get the following error:
```
{
  "errors": [
    {
      "debugMessage": "Expected a value of type \"String\" but received: [{\"fieldId\":\"5\",\"ownerId\":\"31\",\"sortOrder\":\"1\",\"dirty\":false,\"collapsed\":false,\"deletedWithOwner\":false,\"id\":32,\"tempId\":null,\"draftId\":null,\"revisionId\":null,\"uid\":\"eb4ea93d-bb65-497e-a845-61c77e50667e\",\"siteSettingsId\":32,\"fieldLayoutId\":6,\"structureId\":null,\"contentId\":3,\"enabled\":true,\"archived\":false,\"siteId\":1,\"title\":null,\"slug\":null,\"uri\":null,\"dateCreated\":{\"date\":\"2021-01-16 14:10:00.000000\",\"timezone_type\":3,\"timezone\":\"America\\/Los_Angeles\"},\"dateUpdated\":{\"date\":\"2021-01-16 14:10:00.000000\",\"timezone_type\":3,\"timezone\":\"America\\/Los_Angeles\"},\"dateDeleted\":null,\"root\":null,\"lft\":null,\"rgt\":null,\"level\":null,\"searchScore\":null,\"trashed\":false,\"awaitingFieldValues\":false,\"propagating\":false,\"propagateAll\":false,\"newSiteIds\":[],\"resaving\":false,\"duplicateOf\":null,\"previewing\":false,\"hardDelete\":false}]",
      "message": "Internal server error",
      "category": "internal",
      "locations": [
        {
          "line": 13,
          "column": 7
        }
      ],
      "path": [
        "formieSubmissions",
        0,
        "repeater"
      ]
    }
  ],
  "data": {
    "formieSubmissions": [
      {
        "id": "31",
        "emailAddress": null,
        "repeater": null
      }
    ]
  }
}
```

Mutations on the other hand return the following error:
```
{
  "errors": [
    {
      "debugMessage": "Expected a value of type \"String\" but received: []",
      "message": "Internal server error",
      "category": "internal",
      "locations": [
        {
          "line": 5,
          "column": 7
        }
      ],
      "path": [
        "save_contact_Submission",
        "repeater"
      ]
    }
  ],
  "data": {
    "save_contact_Submission": {
      "title": "2021-01-16 14:17:44",
      "emailAddress": "test@test.com",
      "repeater": null
    }
  }
}
```

### Form settings
Please pick from the options below.

- Multi-page form: Yes or No
- Submission Method: Ajax or Page Reload
- Client-side Validation: Yes or No
- Custom Form Templates: Yes or No

**Additional info**
- Plugin version: 1.3.10
- Craft version: Craft Pro 3.5.17.1
- Multi-site: No

**Additional context**

**Comment by engram-design** [comment]
Yep, repeater fields are limited through GQL for the moment, and we'll try to improve that as soon as we can. I just need to investigate some options with the GQL API.

**Comment by danieladarve** [comment]
Thanks for getting back to me @engram-design.

**Comment by engram-design** [comment]
Should be fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.3.21",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.3.22](https://github.com/verbb/formie/releases/tag/1.3.22)
